build.gradle

```diff
@@ -30,7 +30,7 @@ buildscript {
 }
 
 plugins {
-  id 'com.diffplug.spotless' version '5.12.5'
+  id 'com.diffplug.spotless' version '6.9.0'
```

Comment:
This version needed to be updated otherwise spotless would bring resolve the wrong Scala version for scalafmt.